howto.md

```diff
@@ -0,0 +1,206 @@
+| tekster         | ja, minst 1      | Teksten som faktisk vises i varselet med språkkode.                                        | Dersom flere tekster på ulike språk er gitt må én tekst være satt som default | Språkkode må følge ISO-639 (typen `no`, `nb`, `en`...)                                                                                                                                                                      |
+| link            | ikke for beskjed | Lenke som blir aktivert når en person trykker på varselet i varselbjella eller på min side | Komplett URL, inkludert `https` protokoll.                                    |                                                                                                                                                                                                                             |
+| sensitivitet    | ja               | påkrevd level-of-assurance for å kunne se innhold i varsel                                 | Én av `high`, `substantial`                                                   | `high` og `substantial` tilsvarer det som tidligere var henholdvis nivå `4` og `3`. Hvis personen har varsler med sensitivitet `high`, men er logget inn med LoA `substantial`, vil hen se type varsel, men ikke innholdet. |
+| aktivFremTil    | nei              | Tidspunkt for når varslet skal inaktiverer automatisk av systemet                          | Tidspunkt med tidssone. `UTC` eller `Z` er anbefalt                           | Støttes ikke for Innboks-varsler                                                                                                                                                                                            |
+| eksternVarsling | nei              | Om det skal sendes sms og/eller epost til mottaker                                         | Kan kun velge preferert kanal `SMS` eller `EPOST`.                            | Dersom ekstern varslingstekst ikke er satt blir det sendt en standardtekst.                                                                                                                                                 |
+| produsent       | ja               | Teknisk kilde til varsel-eventet                                                           | Ingen spesielle                                                               | Buildere vil forsøke å hente dette automatisk basert på nais-miljøvariabler. Der disse ikke er tilgjengelige må produsent settes manuelt.                                                                                   |
+
+
+### Json-format
```

Comment:
Flytt denne opp før beskrivelse av felter

Reply:
Ikke helt enig. For folk som bruker buildere er ikke json-formatet kjempespennende. Prøvde å sortere info slik at det mer tekniske er lenger ned på siden.